installer/innoinstall.sh

```diff
@@ -0,0 +1,8 @@
+﻿#!/bin/bash
+rm -rf /tmp/inno
+mkdir /tmp/inno
+cd /tmp/inno
+wget -O is.exe http://files.jrsoftware.org/is/5/isetup-5.5.5.exe
+innoextract is.exe
+mkdir -p ~/".wine/drive_c/inno"
+cp -a app/* ~/".wine/drive_c/inno"
```

Comment:
Depending on their shell completion settings, `app/*` might be interpreted as literally `app/*`.


You could do:
```bash
mkdir -p ${HOME}/.wine/drive_c
cp -a /tmp/inno/app ${HOME}/.wine/drive_c/inno
```
This will copy the `/tmp/inno/app` directory to`~/.wine/drive_c/inno` (effectively renaming it)